src/navigation/orbitalnavigator.cpp

```diff
@@ -720,6 +729,121 @@ void OrbitalNavigator::updateCameraStateFromStates(double deltaTime) {
     // Update the camera state
     _camera->setPositionVec3(pose.position);
     _camera->setRotation(composeCameraRotation(camRot));
+
```

Comment:
This part should be moved to somewhere where it works for the path navigator and the future free flight navigator. Move to NavigationHandler?

Reply:
Moved:
![image](https://user-images.githubusercontent.com/4921724/133217130-d3597fee-ee41-4819-a1f8-8144a082ca09.png)